src/Microsoft.AspNet.Mvc.Core/DefaultControllerActionArgumentBinder.cs

```diff
@@ -81,24 +81,21 @@ public class DefaultControllerActionArgumentBinder : IControllerActionArgumentBi
                 operationContext);
 
             var modelBindingResult = await operationContext.ModelBinder.BindModelAsync(modelBindingContext);
-            if (modelBindingResult != null && modelBindingResult.IsModelSet)
+            if (modelBindingResult != null &&
+                modelBindingResult.IsModelSet &&
+                modelBindingResult.ValidationNode != null)
             {
-                var key = modelBindingResult.Key;
                 var modelExplorer = new ModelExplorer(
                     _modelMetadataProvider,
                     metadata,
                     modelBindingResult.Model);
+                var validationContext = new ModelValidationContext(
+                    modelBindingContext.BindingSource,
+                    operationContext.ValidatorProvider,
+                    modelState,
+                    modelExplorer);
 
-                if (modelBindingResult.ValidationNode != null)
```

Comment:
why is this removed? ValidationNode can be null if a model binder chooses not to use a MVN simply because it does not want to validate the model. I expected some tests to break, are they breaking?